flink-ml-parent/pom.xml

```diff
@@ -35,5 +35,6 @@ under the License.
 
 	<modules>
 		<module>flink-ml-api</module>
+		<module>flink-ml</module>
```

Comment:
I think here should be flink-ml-lib and the directory should also be flink-ml-lib rather than flink-ml. It's the artifact id defined in flink-ml-parent/flink-ml/pom.xml and is also the determined module name for ml libraries in FLIP-39.
Maybe all classes in this and related 2 PRs should also be moved from org.apache.flink.ml to  org.apache.flink.ml.lib package.

Reply:
There are two suggestions for the name of ml package
* as the subdirectory of flink-ml-parent，flink-ml is simpler
* the new ml package with pipeline api in spark is named ml

+1 for flink-ml

Reply:
> There are two suggestions for the name of ml package
> 
> * as the subdirectory of flink-ml-parent，flink-ml is simpler
> * the ml package with pipeline api in other tools is often named ml
> 
> +1 for flink-ml

Why the parent module is named flink-ml-parent is to avoid the conflict with flink-libraries/flink-ml in case of confusing users. For more details please refer to the previous discussion in https://github.com/apache/flink/pull/8402 .
In this premise, no modules should be named flink-ml again. If not so, flink-ml is better to use as the name of the parent module rather than that of a sub module.

Reply:
OK, we changed to flink-ml-lib.